src/compiler/parser.ts

```diff
@@ -4887,15 +4891,22 @@ namespace ts {
             return decorators;
         }
 
-        function parseModifiers(): ModifiersArray {
+        function parseModifiers(permitConstAsModifier: boolean = false): ModifiersArray {

```

Comment:
You can skip the default initialization and just make it optional, since you'll just be checking for truthiness